ci-operator/step-registry/gcp/provision/static-creds/gcp-provision-static-creds-ref.yaml

```diff
@@ -0,0 +1,25 @@
+ref:
+  as: gcp-provision-static-creds
+  from_image:
+    namespace: ocp
+    name: "4.15"
+    tag: upi-installer
+  commands: gcp-provision-static-creds-commands.sh
```

Comment:
Need to inject the latest cli via:
```
  cli: latest
```

Reply:
Why do I need the latest cli?

Reply:
Because once `EXTRACT_MANIFEST_INCLUDED` is enabled, it need the latest oc cli.